### ISSUE TITLE
bug: Redshift Security Group Ingress Rule Not Applied

### ISSUE THREAD

**Issue by JRagone** [localstack/localstack#6718]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When I create an ingress rule for a Redshift cluster security group, the rule doesn't stick: I'll describe cluster security groups, and the rule won't be applied.

### Expected Behavior

From [this issue](https://github.com/localstack/localstack/issues/2775), I figured that the rule would be properly applied.

### How are you starting LocalStack?

With a docker-compose file

#### How are you starting localstack (e.g., `bin/localstack` command, arguments, or `docker-compose.yml`)

```sh
services:
  localstack:
    image: localstack/localstack:latest
    ports:
      - 4563-4584:4563-4584
    volumes:
      - ./volume:/var/lib/localstack
```

#### Client commands (e.g., AWS SDK code snippet, or sequence of "awslocal" commands)

```sh
awslocal redshift create-cluster-security-group --cluster-security-group-name "group-name" --description "Security group description"
awslocal redshift authorize-cluster-security-group-ingress --cluster-security-group-name "group-name" --cidrip 192.168.100.101/32
awslocal redshift describe-cluster-security-groups
```

### Environment

```markdown
- OS: Ubuntu 20.04
- LocalStack: latest
```

### Anything else?

This was tested with and without Pro. I'm curious if this should be a Pro-only feature though.

**Comment by whummer** [comment]
Thanks for reporting @JRagone . Looks like I'm unable to replicate this issue with the latest version - getting this output from the last CLI call:

```
$ awslocal redshift describe-cluster-security-groups
{
    "ClusterSecurityGroups": [
        {
            "ClusterSecurityGroupName": "Default",
            "Description": "Default Redshift Security Group",
            "EC2SecurityGroups": [],
            "IPRanges": [],
            "Tags": []
        },
        {
            "ClusterSecurityGroupName": "group-name",
            "Description": "Security group description",
            "EC2SecurityGroups": [],
            "IPRanges": [],
            "Tags": []
        }
    ]
}
```

Can you please do a `docker pull localstack/localstack` to ensure the latest Docker image is downloaded, and also ensure that the `latest` image is configured in your startup configuration (e.g., docker-compose.yml)? Please let us know if that works. Thanks!

**Comment by JRagone** [comment]
Thanks for checking this out @whummer. I get the same output, but I would have expected to see the CIDR address `192.168.100.101/32` in the `IPRanges` field after the command to authorize an ingress rule on the group. Would that not be expected?

**Comment by whummer** [comment]
Ok, thanks for confirming, that was an oversight on my part. Looks like a bug, we'll double check and fix that! 👍

**Comment by whummer** [comment]
Hi @JRagone , this should now be fixed via #6749 - can you please pull the `latest` Docker image and give it another try? Please let us know if the problem persists.. Thanks!
